src/id-card/sign-permission-confirmation.html

```diff
@@ -0,0 +1,196 @@
+<!doctype html>
+<html lang="en-NZ">
+<head>
+    <title>InternetNZ | IRMA examples - ID-Card</title>
+
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <style>
+        [v-cloak] {
+            display: none;
+        }
+    </style>
+
+    <!-- CSS only -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-+0n0xVW2eSR5OomGNYDnhzAbDsOXxcvSN1TPprVMTNDbiYZCxYbOOl7+AMvyTG2x" crossorigin="anonymous">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+
+    <!-- JavaScript Bundle with Popper -->
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.1/dist/js/bootstrap.bundle.min.js" integrity="sha384-gtEjrD/SeCtmISkJkNUaaKMoLD0//ElJ19smozuHV6z3Iehds+3Ulb9Bn9Plx0x4" crossorigin="anonymous"></script>
+
+    <!-- development version, includes helpful console warnings -->
+    <script src="https://unpkg.com/vue@next"></script>
+
+    <script src="../assets/lib/irma-0.3.3.js" defer></script>
+    <script src="../assets/lib/luxon-1.26.0.min.js" defer></script>
+    <script src="../assets/js/irma-functions.js" defer></script>
+    <script src="../assets/js/ui-functions.js" defer></script>
+    <script src="../assets/js/api-service.js" defer></script>
+    <script src="../assets/js/global.js">
+        // --------------------------------
+        // globals loaded from globals file
+        // --------------------------------
+    </script>
+    <script type="text/javascript">
+        window.onload = function () {
+            const IdCardApp = {
+                data() {
+                    return {
+                        CONSENT_MESSAGE: 'I consent IDNZ company to share my name and email with PartyB company.',
```

Comment:
update this to read "I consent IDNZ to share my disclosed name and email with Centrality for the purpose of associating my IDNZ card with a CENNZnet account". I'm just thinking about how we make this feel a bit more useful... I can imagine a use case where your ID could be associated with your blockchain address/account for example.

Reply:
if you can change any other references to Party B in the app too please.

Reply:
So we then naming PartyB as Centrality, right?

I thought Centrality was going to be only our storage method, but that is fine if we can reference them as the Party involved in using the info.

Reply:
This is not a technical thing, it's a scenario to help people understand the point of what we are doing :)